### PR TITLE
Output for version #235

### PR DESCRIPTION
**Pull request by sveneld**
Type: bugfix
Issue: Output for version #235
Breaking change: no

### CODE REVIEW

File: src/main/php/PHPMD/PHPMD.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ class PHPMD
     /**
      * The current PHPMD version.
      */
-    const VERSION = '@project.version@';
+    const VERSION = '@package_version@';
```
Comment (ravage84) [inline src/main/php/PHPMD/PHPMD.php:28]: How does that make sense?

The property is indeed called ``project.version``:

https://github.com/phpmd/phpmd/blob/c7ee4540026ff0dce70527e4beaf7eba6da3da66/build.properties#L4

**Comment by sveneld** [comment]
@tvbeek can you look?

**Comment by sveneld** [comment]
`continuous-integration/travis-ci Expected — Waiting for status to be reported`
 why so long?

**Comment by kylekatarnls** [comment]
A Travis issue, the build succeeded:
https://travis-ci.org/github/phpmd/phpmd/builds/670234459

**Pull request merged**